cypress/plugins/db_setup.ts

```diff
@@ -15,8 +19,30 @@ export default async () => {
       create: TEST_USER as User,
       update: {},
     });
+    await prisma.dept.create({
+      data: {
+        id: TEST_DEPARTMENT_UUID,
+        name: "Software Engineering",
+        abbreviation: "SFWR"
+      }
+    })
+    await prisma.course.create({
+      data: {
+        id: TEST_COURSE_UUID,
+        name: "Very Hard Course",
+        code: "2H03",
+        term: "Winter",
+        year: 2022,
```

Comment:
Please use the latest schema that I merged today. The schema does not contain `year` anymore.